tests/frontend/test_tdmprogram.py

```diff
@@ -451,4 +499,158 @@ class DummyCompiler(sf.compilers.compiler.Compiler):
             ops.MeasureHomodyne(p[1]) | q[i]
 
     with pytest.raises(sf.program_utils.CircuitError, match=match):
-        new_prog = prog.compile(device=spec, compiler=DummyCompiler())
+        new_prog = prog.compile(device=spec, compiler=DummyCircuit())
```

Comment:
This should probably be left as `DummyCompiler`. A bit confusing perhaps when things are being changed in different PRs back and forth. :smile: